app/views/posts/index.html.erb

```diff
@@ -1 +1,39 @@
-<h1>Here is a list of posts for a given user</h1>
+<div class="container">
+  <div class="row mt-5">
+    <div class="col-md-4">
+      <img src="<%= @user.photo %>" alt="User Photo" class="rounded-circle img-fluid">
+    </div>
+    <div class="col-md-8">
+      <h2><%= @user.name %></h2>
+      <p><%= @user.posts_counter %> Posts</p>
+      <p><strong>Bio:</strong> <%= @user.bio %></p>
+    </div>
+  </div>
+  <% @posts.each_with_index do |post,index| %>
+    <div class="card mb-4">
+      <div class="card-body">
+                    <h5 class="card-title">
+              <%= link_to " #{post.title} ##{index+1}", user_post_path(@user, post) %>
+            </h5>
+        <p class="card-text"><%= post.text %></p>
+        <h6>Comments:</h6>
+        <% post.comments.each do |comment| %>
```

Comment:
- [ ] Good job so far 👍🏽 . Kindly use the  @recent_comments to render 5 most recent comments as specified in the project requirement and remember to add the method to the show of your posts controller 👍🏽